lib/kramdown/converter/odin_html.rb

```diff
@@ -17,10 +17,9 @@ def convert_img(element, _indent)
       def convert_a(element, indent)
         if element.attr['href'].starts_with?('http')
           element.attr.merge!(EXTERNAL_LINK_ATTRIBUTES)
-          super(element, indent)
-        else
-          super
         end
+
+        super
```

Comment:
~This doesn't look right, caused by Rubocop fixes. Will investigate~

Actually works as expected as far as I can tell 👌

Reply:
Looks good to me!